Formula/smug.rb

```diff
@@ -0,0 +1,28 @@
+class Smug < Formula
+  desc "Automate your tmux workflow"
+  homepage "https://github.com/ivaaaan/smug"
+  url "https://github.com/ivaaaan/smug/archive/refs/tags/v0.2.5.tar.gz"
+  sha256 "314711f616ea71613fe883fe579e7d3bc0051f540f27c7925a1ce6a0ece69378"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "tmux" => :test
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{version}")
+  end
+
+  test do
+    (testpath/"test.yml").write <<~EOF
+      session: homebrew-test-session
+      windows:
+        - name: test
+    EOF
+
+    system bin/"smug -v"
```

Comment:
```suggestion
    system bin/"smug", "-v"
```
We should at least try to check the output here.